client/rest/package.json

```diff
@@ -7,6 +7,7 @@
     "start": "node src/index.js",
     "lint": "eslint .",
     "test": "mocha --full-trace --recursive ./test",
+    "coverage:test": "nyc --require mocha --reporter=lcov npm run test",
     "lint:jenkins": "eslint -o lint.client.rest.xml -f junit . || exit 0",
     "test:jenkins": "JUNIT_REPORT_PATH=tests.client.rest.xml mocha --reporter mocha-jenkins-reporter --forbid-only --full-trace --recursive ./test || exit 0"
```

Comment:
do you still need test:jenkins ? if not, i'd replace it with coverage:test

Reply:
Yea, can remove test:jenkins

Reply:
i'd replace test:jenkins with coverage:test in that case

Reply:
ok

Reply:
can you just reorder this one too (lint, test, lint:jenkins, test:jenkins)